modules/xmpp/SignalingLayerImpl.js

```diff
@@ -391,4 +401,33 @@ export default class SignalingLayerImpl extends SignalingLayer {
             this._addLocalSourceInfoToPresence();
         }
     }
+
+    /**
+     * @inheritDoc
+     */
+    getTrackSourceName(ssrc) {
+        return this.sourceNames.get(ssrc);
+    }    
+    
+    /**
+     * Saves the source name for a track identified by it's ssrc.
+     * @param {number} ssrc the ssrc of the target track.
+     * @param {SourceName} sourceName the track's source name to save.
+     * @throws TypeError if <tt>ssrc</tt> is not a number
+     */
+    setTrackSourceName(ssrc, sourceName) {
+        if (typeof ssrc !== 'number') {
+            throw new TypeError(`SSRC(${ssrc}) must be a number`);
+        }
+        
+        // Now signaling layer instance is shared between different JingleSessionPC instances, so although very unlikely
+        // an SSRC conflict could potentially occur. Log a message to make debugging easier.
+        if (this.sourceNames.has(ssrc)) {
```

Comment:
lets also check for the value? Jaya recently pushed a change in regards to the SSRC owner handling

Reply:
see https://github.com/jitsi/lib-jitsi-meet/commit/5ae76612b411aefa9c9d3448091feca634ba6687